time_series/time_series_extraction.qmd

```diff
@@ -35,42 +35,34 @@ import os
 import sys
 import subprocess
 
-# GRASS GIS database variables
-grassbin = "grass-dev"
-grassdata = os.path.join(os.path.expanduser('~'), "grassdata")
-project = "eu_laea"
-mapset = "italy_LST_daily"
-
+# Ask GRASS GIS where its Python packages are
 sys.path.append(
-    subprocess.check_output([grassbin, "--config", "python_path"], text=True).strip()
+    subprocess.check_output(["grass", "--config", "python_path"], text=True).strip()
 )
-```
-
-```{python}
-#| echo: false
-
 # Import the GRASS GIS packages we need
 import grass.script as gs
 import grass.jupyter as gj
 
+path_to_project = "eu_laea/italy_LST_daily"
```

Comment:
```suggestion
path_to_project = "italy_eu_laea"
```

Reply:
This is the project name in the new dataset. As discussed eventually I removed the grassdata level above project.